src/payment/strategies/paypal-commerce/paypal-commerce-script-loader.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('PaypalCommerceScriptLoader', () => {
                 .mockImplementation((url: string) => {
                     (window as PaypalCommerceHostWindow).paypal = paypal;
 
-                    ['client-id=aaa', 'currency=EUR', 'disable-funding=credit', 'intent=capture' ].forEach(str => {
+                    ['client-id=aaa', 'currency=EUR', 'disable-funding=credit,card', 'intent=capture' ].forEach(str => {
```

Comment:
one extra whitespace at the very end of the array 🍹

Reply:
Thank you!